src/components/Column/Column.tsx

```diff
@@ -13,23 +14,22 @@ export interface ColumnProps {
   column: ColumnDefinition;
   draggable?: boolean;
   onEditClick?: () => void;
-  onAddTicket?: (value: string) => void;
   className?: string;
   style?: CSSProperties;
 }
 
-export default function Column({
-  column,
-  draggable,
-  onEditClick,
-  onAddTicket,
-  className,
-  style,
-}: ColumnProps): JSX.Element {
-  const { call } = useMutation<ColumnDefinition>(UPDATE_COLUMN_TITLE(1, column.id), "PATCH");
+export default function Column({ column, draggable, onEditClick, className, style }: ColumnProps): JSX.Element {
+  const { call: updateColumnTitle } = useMutation<ColumnDefinition>(UPDATE_COLUMN_TITLE(1, column.id), "PATCH");
+  const { call: createTicket } = useMutation<TicketDefinition>(CREATE_TICKET(1, column.id), "POST");
 
   const handleTitleBlur = async (value: string) => {
-    await call({
+    await updateColumnTitle({
+      variables: { name: value },
+    });
+  };
+
+  const handleAddTicket = async (value: string) => {
+    await createTicket({
       variables: { name: value },
```

Comment:
our current implementation of createTicket require `sort: number` but we should fix backend side.